pep8.py

```diff
@@ -1056,17 +1057,61 @@ def is_binary_operator(token_type, text):
         if ('\n' in text or '\r' in text) and token_type != tokenize.STRING:
             line_break = True
         else:
-            if (is_binary_operator(token_type, text) and line_break and
+            if (_is_binary_operator(token_type, text) and line_break and
                     not unary_context and
-                    not is_binary_operator(previous_token_type,
-                                           previous_text)):
+                    not _is_binary_operator(previous_token_type,
+                                            previous_text)):
                 yield start, "W503 line break before binary operator"
             unary_context = text in '([{,;'
             line_break = False
             previous_token_type = token_type
             previous_text = text
 
 
+def break_after_binary_operator(logical_line, tokens):
```

Comment:
So let me give you a practical example, let's say we have a function that takes a callable and yields data and it looks like `gen_func_0(filter=None)`. Let's say we have a common filter that we always want to apply and just return what's there, that would be a good use case for another function, e.g., `gen_func_1()` which wants to yield data yielded by `gen_func_0`. In Python 2, `gen_func_1` is implemented like so:

```py
def gen_func_1():
    for result in gen_func_0(common_filter):
        yield result
```

or 

```py
def gen_func_1():
    return (r for r in gen_func_0(common_filter))
```

The latter is shorter, but neither is really elegant. In Python 3, you could instead do:

```py
def gen_func_1():
    yield from gen_func_0(common_filter)
```

This means that in a magical future where that function becomes part of an asyncio application you can also then decorate it as something that is a co-routine (which doesn't help us in this very case, but makes it a good feature to start using).